rtos/rtx/TARGET_CORTEX_M/RTX_CM_lib.h

```diff
@@ -361,6 +361,8 @@ extern void pre_main (void);
 static uint32_t thread_stack_main[DEFAULT_STACK_SIZE / sizeof(uint32_t)];
 #elif defined(TARGET_XDOT_L151CC)
 static uint32_t thread_stack_main[DEFAULT_STACK_SIZE * 6 / sizeof(uint32_t)];
+#elif defined(TARGET_REALTEK_RTL8195AM)
```

Comment:
Yes @bulislaw , only SPI test uses that much of main stack memory and cause the trouble.    
1. the CI test put all its objects as function variables so they are put in main stack memory. If user put this object in another thread then I think there will be no problem.  
2. Our GDMA implementation uses relative larger size of memory compared to other peripherals, but it might take much effort to modify and debug it.